<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_pesquisar_produto</class>
 <widget class="QDialog" name="fm_pesquisar_produto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pesquisar produtos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/imagens/img/images.png</normaloff>:/imagens/img/images.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_pp_produtos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>581</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Filtrar por:</string>
   </property>
   <widget class="QRadioButton" name="rb_pp_codigo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Codigo</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_pp_produto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Produto</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Filtar:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="txt_pp_filtrar"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_pp_filtrar">
        <property name="text">
         <string>ok</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
